tabpy-server/server_tests/test_pwd_file.py

```diff
@@ -0,0 +1,107 @@
+import logging
+import os
+import unittest
+from argparse import Namespace
+from tempfile import NamedTemporaryFile
+
+from tabpy_server.app.app import TabPyApp
+from tabpy_server.app.ConfigParameters import ConfigParameters
+
+try:
+    from unittest.mock import patch, call
+except ImportError:
+    from mock import patch, call
+
+
+class TestPasswordFile(unittest.TestCase):
+    def __init__(self, *args, **kwargs):
+        super(TestPasswordFile, self).__init__(*args, **kwargs)

```

Comment:
While I would suggest getting rid of init altogether just calling out that you don't need to call the super in this fashion anymore as this is more of a Py2 compat call. Instead could just do `super().__init(*args, **kwargs)`

Reply:
fixing with the next push